oc-aliases.sh

```diff
@@ -80,29 +87,29 @@ function load_connection() {
 		echo "load_connection <connection.json>"
 	else
 		gettoken
-		curl -k -H "Authorization: Bearer ${TOKEN}" https://${SD_HOST:?environment variable must be set}/connmgr/v1/connections -X POST -H "Content-type: application/json" -d@"$1"
+		curl -k -H "Authorization: Bearer ${TOKEN}" https://${SD_CONSOLE_ROUTE:?environment variable must be set}/connmgr/v1/connections -X POST -H "Content-type: application/json" -d@"$1"
```

Comment:
Once defined SD_CONSOLE_ROUTE, you can remove the ":?environment variable must be set"